### PR TITLE
History-Page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Added History Page in Video History.
Screenshot (368)

### CODE REVIEW

File: src/Pages/History-page/Components/Header.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.history-header--container {
+  z-index: 1;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0.2rem 0 0 0.2rem;
```
Comment (fahd23) [inline src/Pages/History-page/Components/Header.css]: you can use px here as it is less than 10px

File: src/Pages/History-page/Components/HistoryCard.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        history.map((item) => {
+          return (
+            <div key={item._id} className="card--wrapper">
+              <div className="card--thumbnail">
+                <img
+                  className="card__img--thumbnail"
+                  src={item.thumbnail}
+                  onClick={() => {
+                    dispatch({ type: "Add_to_CurrentVideo", payload: item });
+                    dispatch({ type: "Add_to_History ,", payload: item });
```
Comment (fahd23) [inline src/Pages/History-page/Components/HistoryCard.jsx]: `type: "Add_to_History ,"` , do u need comma here?

**Review by fahd23 — APPROVED** [review]
Given Some suggestions , code looks fine to me

File: src/Pages/History-page/Components/HistoryCard.jsx (hunk 2 above)
Comment (t007rushi) [reply src/Pages/History-page/Components/HistoryCard.jsx]: rightly pointed out by @fahd23 I think its typo

File: src/Pages/History-page/History.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import "../Home-Page/Home-Responsive.css";
+import { Nav, NavForTab, Sidebar } from "../../components/Index";
+import { HistoryCard } from "./Components/HistoryCard.jsx";
+import { Header } from "./Components/Header";
+export function History() {
+  return (
+    <div className="page--Container">
+      <Nav hideComponent={false} />
+      <NavForTab />
+      <Sidebar />
```
Comment (t007rushi) [inline src/Pages/History-page/History.jsx]: Suggestion : you separate out Layout and components for clean structure for file management

**Review by t007rushi — APPROVED** [review]
Look for suggestion by fahd and me 
else go ahead

File: src/Pages/History-page/Components/HistoryCard.jsx (hunk 2 above)
Comment (SaurabhChawda) [reply src/Pages/History-page/Components/HistoryCard.jsx]: Updated ThankYou

File: src/Pages/History-page/History.jsx (hunk 3 above)
Comment (SaurabhChawda) [reply src/Pages/History-page/History.jsx]: Updated ThankYou

**Pull request merged**